src/encord_active/server/routers/project.py

```diff
@@ -56,10 +75,32 @@ def read_item_ids(
     ids: Annotated[Optional[list[str]], Body()] = None,
 ):
     merged_metrics = filtered_merged_metrics(project, filters)
-    column = [col for col in merged_metrics.columns if col.lower() == sort_by_metric.lower()][0]
-    res = merged_metrics[[column]].dropna().sort_values(by=[column], ascending=ascending)
+
+    try:
+        column = [col for col in merged_metrics.columns if col.lower() == sort_by_metric.lower()][0]
+        df = merged_metrics
+    except:
+        try:
+            if filters.prediction_filters is None:
+                raise
+            df, _ = get_model_predictions(project, filters.prediction_filters)
+
+            if filters.prediction_filters.outcome == ObjectDetectionOutcomeType.FALSE_NEGATIVES:
+                sort_by_metric = re.sub(r"(.*?\()P(\))", r"\1O\2", sort_by_metric)
```

Comment:
Is this really `\10` always?
I understand it as getting a metric like "Object Area (P)" and then returning the same thing, but may be missing something?

Reply:
This should work for all false negative metrics, it just replaces the `P` with an `O`. I'll replace the regex with a simpler `str.replace` which will make this part more explicit.

Reply:
Hahaha my bad 🤦🏻